bokeh/application/handlers/code_runner.py

```diff
@@ -118,6 +122,13 @@ def __init__(self, source: str, path: PathLike, argv: List[str], package: Module
 
     # Properties --------------------------------------------------------------
 
+    @property
+    def doc(self) -> str | None:
+        ''' Contents of docstring, if code contains one.
+
+        '''
+        return self._doc
+
```

Comment:
This would be amenable to a unit test in https://github.com/bokeh/bokeh/blob/branch-2.4/tests/unit/bokeh/application/handlers/test_code_runner.py

Reply:
Test added.